pom.xml

```diff
@@ -130,9 +130,9 @@
     <glusterfs-hadoop.version>2.3.13</glusterfs-hadoop.version>
     <grpc.version>1.37.0</grpc.version>
     <gson.version>2.8.9</gson.version>
-    <netty.version>4.1.52.Final</netty.version>
+    <netty.version>4.1.86.Final</netty.version>
```

Comment:
netty is one of the delicate dependencies in the product that could have potential ramifications. we would not be able to take this specific change without extensive testing, in particular around the areas of scale and performance. i would advise to exclude this change from this PR